code/modules/mob/living/carbon/human/species_types/android.dm

```diff
@@ -2,27 +2,64 @@
 	name = "Android"
 	id = "android"
 	say_mod = "states"
-	species_traits = list(NOBLOOD, NO_DNA_COPY, NOTRANSSTING)
-	inherent_traits = list(TRAIT_RESISTHEAT,TRAIT_NOBREATH,TRAIT_RESISTCOLD,TRAIT_RESISTHIGHPRESSURE,TRAIT_RESISTLOWPRESSURE,TRAIT_RADIMMUNE,TRAIT_GENELESS,TRAIT_NOFIRE,TRAIT_PIERCEIMMUNE,TRAIT_NOHUNGER,TRAIT_LIMBATTACHMENT)
+	species_traits = list(NOBLOOD, NOZOMBIE, NOHUSK, NO_DNA_COPY, NOTRANSSTING)
+	inherent_traits = list(TRAIT_RESISTHEAT,TRAIT_COLDBLOODED,TRAIT_NOBREATH,TRAIT_RESISTCOLD,TRAIT_RESISTHIGHPRESSURE,TRAIT_RESISTLOWPRESSURE,TRAIT_RADIMMUNE,TRAIT_NOCLONE,TRAIT_TOXIMMUNE,TRAIT_GENELESS,TRAIT_NOFIRE,TRAIT_PIERCEIMMUNE,TRAIT_NOHUNGER,TRAIT_LIMBATTACHMENT,TRAIT_MEDICALIGNORE)
 	inherent_biotypes = list(MOB_ROBOTIC, MOB_HUMANOID)
-	meat = null
+	meat = /obj/item/stack/sheet/plasteel{amount = 5}
+	skinned_type = /obj/item/stack/sheet/metal{amount = 10}
 	damage_overlay_type = "synth"
+	mutantbrain = /obj/item/organ/brain/positron
+	mutanteyes = /obj/item/organ/eyes/robotic
 	mutanttongue = /obj/item/organ/tongue/robot
+	mutantliver = /obj/item/organ/liver/cybernetic/upgraded/ipc
+	mutantstomach = /obj/item/organ/stomach/cybernetic	//Is there even anything an android would use a stomach for?
+	mutantears = /obj/item/organ/ears/cybernetic
 	species_language_holder = /datum/language_holder/synthetic
 	limbs_id = "synth"
+	toxmod = 0
+	clonemod = 0
+	reagent_tag = PROCESS_SYNTHETIC
+	species_gibs = "robotic"
+	attack_sound = 'sound/items/trayhit1.ogg'
+	screamsound = 'goon/sound/robot_scream.ogg'
+	allow_numbers_in_name = TRUE
+	deathsound = 'sound/voice/borg_deathsound.ogg'
+	wings_icon = "Robotic"
 	changesource_flags = MIRROR_BADMIN | MIRROR_MAGIC | WABBAJACK | RACE_SWAP | ERT_SPAWN | SLIME_EXTRACT
 
 /datum/species/android/on_species_gain(mob/living/carbon/C)
 	. = ..()
 	for(var/X in C.bodyparts)
 		var/obj/item/bodypart/O = X
 		O.change_bodypart_status(BODYPART_ROBOTIC, FALSE, TRUE)
+	
+	for (var/obj/item/bodypart/BP in C.bodyparts)
+		BP.burn_reduction = 5
+		BP.brute_reduction = 4
+		
+		BP.light_brute_msg = "marred"
+		BP.medium_brute_msg = "dented"
+		BP.heavy_brute_msg = "falling apart"
+		BP.light_burn_msg = "scorched"
+		BP.medium_burn_msg = "charred"
+		BP.heavy_burn_msg = "smoldering"
```

Comment:
this shouldn't be needed since the limbs are already getting replaced by robotic ones

Reply:
When I commented that out, when I inspect myself for wounds it tells me "bruised" instead of "marred."

Reply:
... that's odd, preterni show the proper messages, maybe tied to ROBOTIC_LIMBS then.
in that case, just change it to use the defines rather than test (ROBOTIC_LIGHT_BRUTE_MSG and the like)

Reply:
I don't think it works with preternis actually, if you click on your health doll when injured it will say bruised for limbs with brute damage.

Reply:
> ... that's odd, preterni show the proper messages, maybe tied to ROBOTIC_LIMBS then. in that case, just change it to use the defines rather than test (ROBOTIC_LIGHT_BRUTE_MSG and the like)

If I make it use stuff like ROBOTIC_LIGHT_BRUTE_MSG, I get an error on compiling for an undefined var, should I define them at the top of the file or just not use the defines? Or is there some other thing I should be doing?

Reply:
Oh, if preternis have the wrong text then leave that issue for a different PR as it means there is an underlying issue.

Reply:
Should I revert this commit then? Or leave it?

Reply:
probably revert it

Reply:
Reverted